packages/core/addon/models/bard-request-v2/fragments/base.js

```diff
@@ -21,4 +33,38 @@ export default class Base extends Fragment.extend(Validations) {
     }
   })
   parameters;
+  @attr('string') type;
+
+  @service('bard-metadata')
+  metadataService;
+
+  /**
+   * Datasource or namespace
+   * @property {string}
+   */
+  source;
+
+  /**
+   * @type {Meta}
+   */
+  @computed('field', 'type', 'source')
+  get columnMeta() {
+    assert('Source must be set in order to access columnMeta', isPresent(this.source));
+    assert('column type must be set in order to access columnMeta', isPresent(this.type));
+    if (this.lookupField === 'dateTime') {
+      return {
+        id: 'dateTime',
+        name: 'Date Time'
+      };
+    }
+    return this.metadataService.getById(this.type, this.lookupField, this.source);
+  }
+
+  /**
+   * Field without dimension subfield
+   * @type {string}
+   */
+  get lookupField() {
+    return this.field.split('.')[0];
```

Comment:
I know Elide uses `${tableName}.{columnName}` for its ids, we might want to rethink how we handle Fili dimension fields